packages/eslint-plugin-wdio/lib/index.ts

```diff
@@ -0,0 +1,28 @@
+import awaitExpect from './rules/await-expect'
+import noDebug     from './rules/no-debug'
+import noPause     from './rules/no-pause'
+
+const rules = {
+    'await-expect' : awaitExpect,
+    'no-debug'     : noDebug,
+    'no-pause'     : noPause
+}
+
+const configs = {
+    recommended: {
+        globals: {
+            $                  : false,
+            $$                 : false,
+            browser            : false,
+            driver             : false,
+            expect             : false,
+            multiremotebrowser : false,
+        },
+        rules : {}
```

Comment:
Shouldn't the rules imported above be recommended?

Reply:
Yeah probably, I'll create a branch that does that along with adding some docs for it.

Reply:
I added this in https://github.com/webdriverio/webdriverio/pull/8095